src/test/java/uk/gov/justice/digital/converter/dms/DMS_3_4_6_ConverterTest.java

```diff
@@ -1,6 +1,9 @@
 package uk.gov.justice.digital.converter.dms;
 
+import com.codahale.metrics.MetricRegistryListener;
 import lombok.val;
+import org.apache.spark.sql.SparkSession;
+import org.junit.jupiter.api.BeforeAll;
```

Comment:
None of these added imports are used so can be removed.

Reply:
removed